code/style.css

```diff
@@ -1,140 +1,158 @@
+}
+
+input,
+select {
+	box-sizing: border-box;
+	border: none;
+	border-radius: 4px 0 0 4px;
+	background: #e5e9ff;
+	color: #0026ff;
+	padding: 16px;
+	font-size: 16px;
+	font-family: 'Montserrat';
+	font-weight: 600;
+	line-height: 26px;
+	flex: 1;
+
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	appearance: none;
 }
 
 main {
-  margin: 0 auto;
-  width: 100%;
-  max-width: 700px;
-  height: 600px;
-  border-radius: 30px;
-  background: #fff;
-  padding: 20px 24px;
-  padding-top: 0;
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
+	margin: 0 auto;
+	width: 100%;
+	max-width: 700px;
+	height: 600px;
+	border-radius: 30px;
+	background: #fff;
+	padding: 20px 24px;
+	padding-top: 0;
+	box-sizing: border-box;
+	display: flex;
+	flex-direction: column;
 }
 
 .chat {
-  flex: 1;
-  display: flex;
-  justify-content: flex-start;
-  overflow: scroll;
-  flex-direction: column;
-  padding-bottom: 16px;
+	flex: 1;
+	display: flex;
+	justify-content: flex-start;
+	overflow: scroll;
+	flex-direction: column;
+	padding-bottom: 16px;
 }
 
 .bot-msg {
-  display: flex;
-  margin: 16px 8px 0 0;
-  flex-shrink: 0;
+	display: flex;
+	margin: 16px 8px 0 0;
+	flex-shrink: 0;
 }
 
 .user-msg {
-  display: flex;
-  justify-content: flex-end;
-  margin: 16px 0 0 8px;
-  flex-shrink: 0;
+	display: flex;
+	justify-content: flex-end;
+	margin: 16px 0 0 8px;
+	flex-shrink: 0;
 }
 
 .bot-msg img,
 .user-msg img {
-  width: 60px;
-  height: 60px;
+	width: 50px;
+	height: 50px;
 }
 
 .bubble {
-  background: #e5e9ff;
-  font-weight: 600;
-  font-size: 16px;
-  line-height: 26px;
-  padding: 16px 24px;
-  color: #0026ff;
-  max-width: 40%;
+	background: #e5e9ff;
+	font-weight: 600;
+	font-size: 16px;
+	line-height: 26px;
+	padding: 16px 24px;
+	color: #0026ff;
+	max-width: 80%;
 }
 
 .bot-bubble {
-  border-radius: 0px 26px 26px 26px;
-  margin-left: 8px;
+	border-radius: 0px 26px 26px 26px;
+	margin-left: 8px;
 }
 
 .user-bubble {
-  border-radius: 26px 0 26px 26px;
-  margin-right: 8px;
+	border-radius: 26px 0 26px 26px;
+	margin-right: 8px;
 }
 
 .input-wrapper {
-  display: flex;
-  justify-content: center;
+	display: flex;
+	justify-content: center;
 }
 
 .input-wrapper form {
-  width: 100%;
-  display: flex;
+	width: 100%;
+	display: flex;
 }
 
 button {
-  background-color: #0026ff;
-  color: white;
-  border: none;
-  border-radius: 4px;
-  padding: 16px 40px;
-  margin-right: 4px;
-  font-size: 16px;
-  line-height: 26px;
-  font-family: 'Montserrat';
-  font-weight: 500;
-  cursor: pointer;
-  transition: all 0.3s ease;
+	background-color: #0026ff;
+	color: white;
+	border: none;
+	border-radius: 4px;
+	padding: 16px 30px;
+	margin-right: 4px;
+	font-size: 16px;
+	line-height: 26px;
+	font-family: 'Montserrat';
+	font-weight: 500;
+	cursor: pointer;
+	transition: all 0.3s ease;
 }
 
 button:hover {
-  opacity: 0.9;
-  transition: all 0.2s ease;
+	opacity: 0.9;
+	transition: all 0.2s ease;
+}
+
+@media (min-width: 786px) {
```

Comment:
Smart to have this. It occured to me afterwards when I showed our bot on my cell phone. It would have been nice to add some @media..